modules/jackson/src/main/scala/com/github/eikek/calev/jackson/CalEventSerializer.java

```diff
@@ -0,0 +1,32 @@
+package com.github.eikek.calev.jackson;
```

Comment:
Maybe the java files can better go into `src/main/java` source dir? Or other question: why not use Scala? :-) Can't it be done using Scala for some reason?

Reply:
Good point 👍🏻